server/index.ts

```diff
@@ -122,6 +136,35 @@ async function start(id: number, disconnect: () => void) {
       }`
     );
   });
+  server.on("shutdown", () => {
+    // This is done so that new rate limiter
+    // configurations would immediately come into effect
+    // on server restart. As an example,
+    // the `duration` property doesn't change
+    // for already created keys, but affects all new keys.
+    //
+    // Refer https://github.com/animir/node-rate-limiter-flexible/wiki/Options#clearexpiredbytimeout
+    Logger.info("lifecycle", `Deleting all rate limiter specific keys...`);
+
+    const scanStream = Redis.defaultClient.scanStream({
+      match: "rlflx:*",
```

Comment:
`rlflx` should be a constant as it's loosely coupled here that all the keys begin with this, why "rlflx" btw?

Reply:
No particular reason. I borrowed it from [here](https://github.com/animir/node-rate-limiter-flexible/wiki/Options#keyprefix)(looks like it's a shorter form of `node-rate-limiter-flexible` :smile: ). We should change it since it doesn't convey much about itself. Should we make it `rate_limiter`?

Reply:
Maybe just `rl:` is fine, every byte counts in keeping redis memory usage low

Reply:
Fixed!